mollie/api/resources/permissions.py

```diff
@@ -0,0 +1,16 @@
+from ..error import IdentifierError
+from ..objects.permission import Permission
+from .base import Base
+
+
+class Permissions(Base):
+
+    def get_resource_object(self, result):
+        return Permission(result, self.client)
+
+    def get(self, permission_id, **params):
+        if not permission_id:
```

Comment:
Based on https://docs.mollie.com/oauth/permissions, I would think that it's easy to get a better validation on the input than boolean. Maybe a regex for "lowercase chars + dot + lowercase chars"? Additionally, we should that under test.

Reply:
Yes, done!

Reply:
nice